cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -1684,11 +1682,14 @@ subroutine makemask
          tarean(:,:,iblk) = c0
          tareas(:,:,iblk) = c0
 
-         do j = 1, ny_block
-         do i = 1, nx_block
+         do j = jlo,jhi
```

Comment:
Are the ghost cells for these masks filled later?

Reply:
They are not, but the halos are never accessed, so there is no error when we initialize with snan.  There are probably many arrays that have uninitialized memory, but that's fine.  In many ways, that's what we want.  We want to only initialize memory that we think is used.  That way we can trap code (or learn more about our codes) when an snan actually triggers an error.  If we initialized everything explicitly to zero just for the sake of doing so, the snan testing wouldn't tell us anything.